core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1316,58 +1316,57 @@ public MetricKey build() {
+              + "without external RPC involved. Data exists in worker storage "
+              + "or is fetched by workers from UFSes. This records data read "
               + "by worker internal calls (e.g. clients embedded in workers).")
           .setMetricType(MetricType.GAUGE)
           .build();
   public static final MetricKey CLUSTER_BYTES_READ_REMOTE =
       new Builder("Cluster.BytesReadRemote")
-          .setDescription("Total number of bytes read from Alluxio storage "
-              + "or underlying UFS if data does not exist in Alluxio storage "
-              + "reported by all workers. This does not include "
-              + "short-circuit local reads and domain socket reads")
+          .setDescription("Total number of bytes read from all workers via network (RPC). "
+              + "Data exists in worker storage or is fetched by workers from UFSes. "
+              + "This does not include short-circuit local reads and domain socket reads")
           .setMetricType(MetricType.COUNTER)
           .build();
   public static final MetricKey CLUSTER_BYTES_READ_REMOTE_THROUGHPUT =
       new Builder("Cluster.BytesReadRemoteThroughput")
-          .setDescription("Bytes read per minute throughput from Alluxio storage "
-              + "or underlying UFS if data does not exist in Alluxio storage "
-              + "reported by all workers. This does not include "
+          .setDescription("Bytes read per minute throughput from all workers "
+              + "via network (RPC calls). Data exists in worker storage "
+              + "or is fetched by workers from UFSes. This does not include "
               + "short-circuit local reads and domain socket reads")
           .setMetricType(MetricType.GAUGE)
           .build();
   public static final MetricKey CLUSTER_BYTES_READ_DOMAIN =
       new Builder("Cluster.BytesReadDomain")
-          .setDescription("Total number of bytes read from Alluxio storage "
-              + "via domain socket reported by all workers")
+          .setDescription("Total number of bytes read from all works "
+              + "via domain socket")
           .setMetricType(MetricType.COUNTER)
           .build();
   public static final MetricKey CLUSTER_BYTES_READ_DOMAIN_THROUGHPUT =
       new Builder("Cluster.BytesReadDomainThroughput")
-          .setDescription("Bytes read per minute throughput from Alluxio storage "
-              + "via domain socket by all workers")
+          .setDescription("Bytes read per minute throughput from all workers "
+              + "via domain socket")
           .setMetricType(MetricType.GAUGE)
           .build();
   public static final MetricKey CLUSTER_BYTES_READ_LOCAL =
       new Builder("Cluster.BytesReadLocal")
-          .setDescription("Total number of bytes short-circuit read from local storage "
+          .setDescription("Total number of bytes short-circuit read from local worker data storage "
```

Comment:
It is a little weird to me that it's an aggregated number, but we say "local worker".